src/mds/mdstypes.h

```diff
@@ -470,6 +470,7 @@ struct inode_t {
 
   // affected by any inode change...
```

Comment:
The above comment will be confusing after this, maybe add something like:

// set to ctime when file is created, and only changes afterword with explict setattr